testing/scenario_app/ios/Scenarios/Scenarios/Info.plist

```diff
@@ -20,6 +20,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>UILaunchStoryboardName</key>
```

Comment:
For some reason if I take this line out, on iOS 13 `XCUIScreenshot` captures the status bar.